db/db_impl_open.cc

```diff
@@ -518,6 +531,37 @@ Status DBImpl::Recover(
   return s;
 }
 
+Status DBImpl::CreatePersistStatsHandle() {
```

Comment:
This function name is misleading to me. In some cases, it's not just to create the handle but the column family too.